src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const COST_PER_DAY = 40;
+  const TOTAL_COST = days * COST_PER_DAY;
```

Comment:
```suggestion
  const totalCost = days * COST_PER_DAY;
```
it's not a constant

Reply:
exactly! There may be a different number of days.